10_datastructures_lab/01.py

```diff
@@ -1,10 +1,31 @@
-"""
-Write a program that takes two strings
-from the user and checks if they represent
-a valid user name.
-Valid users and passwords:
-    apple => red
-    lettuce => green
-    lemon => yellow
-    orange => orange
-"""
+"""
+excersize #1
+Print IP address of hosts
+"""
+
+import sys
+
+hosts = {}
+
+with open("G:\People\Moshe\hosts.txt", "r") as fin:
+    lines = [line.rstrip('\n') for line in fin]

```

Comment:
That's just overusing list comprehension :)
The down side is you read the file twice (taking twice as long), so better to rstrip() inside the second loop